docs/Maintainer-Guidelines.md

```diff
@@ -104,3 +104,14 @@ Whitespace corrections (to Ruby standard etc.) are allowed (in fact this
 is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
+
+## Communication
+Maintainers have a variety of ways to communicate with each other:
+
+- Homebrew's public repositories on GitHub
+- Homebrew's Slack/Discourse/GitHub repositories/other private communications
+- Homebrew's Slack/Discourse/IRC/other direct 1:1 messages
+
+All communication should ideally occur in public on GitHub. Where this is not possible (e.g. a security disclosure, interpersonal issue between two maintainers, urgent breakage that needs resolved) this move to maintainers' private communication and, if necessary, 1:1 communication. Technical decisions should not happen in 1:1 communications and all decisions made in a private communication should end up back as something linkable in GitHub.
```

Comment:
that needs -> that needs to be

Reply:
"this move to maintainers' private communication" <- some words are missing here

Reply:
>Technical decisions should not happen in 1:1 communications and all decisions made in a private communication should end up back as something linkable in GitHub.

This sentence contradicts itself.

Reply:
@ilovezfs Fixed to clarify that if they happen (or have happened) they should end up on GitHub.